src/views/InteractiveMap/ReactionDialog.vue

```diff
@@ -191,12 +191,12 @@ export default Vue.extend({
     },
     direction() {
       if (this.lowerBound >= 0) {
-        return " -> ";
+        return " 🠖 ";
       }
       if (this.upperBound <= 0) {
-        return " <- ";
+        return " 🠔 ";
```

Comment:
These one-way arrows aren't displayed on Windows8.1 (they come from unicode update in 2014)
I would choose something within U+FFFF range
https://www.compart.com/en/unicode/search?q=leftwards+arrow#characters